lib/octicons_react/src/get-svg-props.js

```diff
@@ -0,0 +1,36 @@
+const sizeMap = {
+  small: 16,
+  medium: 32,
+  large: 64
+}
+
+export default function getSvgProps({ariaLabel, className, size, verticalAlign, svgDataByHeight}) {
```

Comment:
Should this prop be `aria-label` instead of `ariaLabel` . While we're making breaking changes it seems like this could be another spot to clean up and be more consistent with other components in Primer/components.

Reply:
Yeah., I agree that `aria-label` is the correct prop name to be consistent with the rest of React. I was a little worried about making that breaking change because it could break aria-labels without people noticing. However, now is probably a good time to rip off the bandaid and correct this API since we're already making other breaking changes. Okay, you convinced me. I'll change it 😸

Reply:
I'll make that change in a follow up PR though since it's not directly related to 24px icons.